subprojects/core/src/integTest/groovy/org/gradle/api/tasks/ReproducibleArchivesIntegrationTest.groovy

```diff
@@ -125,7 +123,7 @@ class ReproducibleArchivesIntegrationTest extends AbstractIntegrationSpec {
 
         where:
         compression | md5
-        'gzip'      | 'a9339a2b2bb7f96057c480834d00e29e'
+        'gzip'      | (JavaVersion.current().isCompatibleWith(JavaVersion.VERSION_16) ? 'c4d89909b123359774c0a5dfd3cc8e46' : 'a9339a2b2bb7f96057c480834d00e29e')
```

Comment:
Do you know why this is different?

Reply:
ah, this is in the 16 release notes: https://jdk.java.net/16/release-notes#JDK-8244706

Could you add this as a comment and merge this?

Reply:
Thanks! I had left this as a draft with the intention to figure this out before pushing it forward.
Will add the comment and merge.